### DISCUSSION TITLE
e27 camera (CanmSmart) #1336 - device just added in latest update, but 'no support for device'

### DISCUSSION THREAD

**Discussion by phatrickk** [make-all/tuya-local#1779] [Q&A]
Support was just added for the this device: https://github.com/make-all/tuya-local/issues/1336

Huge thanks for getting to this, really appreciate the work that you put into this!

Unfortunately, when trying to add either of my two cameras I'm running into issues. It's very challenging to get the integration to connect to them at all, describing the device as unreachable. And the same occurs when polling the devices with TinyTuya - "no response".

It seems I'm able to connect to the device relatively reliably right after resetting it. Which is true of both the integration and tiny tuya polling. But when the integration does connect (providing UUID, key, Static IP, and protocol) I get the error that the device is not supported.

Did some seaarching here but to no avail. Can find lots of people with connectivity issues to devices, but not sure that's actually my problem as I seem to be able to connect sometimes.

Any ideas?

**Comment by make-all** [comment]
The log in the original report had the device only reporting one dp.  If that dp is not consistent, then it is impossible to recognise the device.

Also, is this battery powered? It sounds like it is acting like a battery powered device (going offline to save battery)
